src/Types.jl

```diff
@@ -309,6 +309,7 @@ end
 # ENV variables to set some of these defaults?
 Base.@kwdef mutable struct Context
     env::EnvCache = EnvCache()
+    io::IO = stdout
```

Comment:
I think we should change this to `stderr`.

Reply:
Please explain. Its not weird to output to `stderr` by default? What is the benefit over `stdout`?

Reply:
I don't think it is that weird. Many unix applications work by piping their stdout to each other. If an application starts spewing loginfo to stdout then that breaks. The loggers in Base already use stderr so I don't think it is weird for us to also log to that.

Reply:
But e.g. for `Pkg.status` it would make sense with `stdout`, no?

Reply:
Tbh, I'm not sure.

Reply:
What would make sense for `Pkg.status` to do would be to print nothing and return some kind of object with the status information. Then the REPL could have it printed the usual way with the `show` method, the Pkg REPL mode could print it any way it finds useful, and a program could look into the object and draw whatever conclusions it finds meaningful.